src/pages/404.js

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+
+const Custom404 = () => {
```

Comment:
@i001962 keen to get this 404 in. Do you wanna update this PR so it just includes for 404 addition?